src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -73,9 +73,9 @@ sealed override static void M10()
                 // (4,26): error CS8703: The modifier 'abstract' is not valid for this item in C# 9.0. Please use language version 'preview' or greater.
                 //     abstract static void M01()
                 Diagnostic(ErrorCode.ERR_InvalidModifierForLanguageVersion, "M01").WithArguments("abstract", "9.0", "preview").WithLocation(4, 26),
-                // (7,25): error CS0112: A static member cannot be marked as 'virtual'
+                // (7,25): error CS8703: The modifier 'virtual' is not valid for this item in C# 9.0. Please use language version 'preview' or greater.
```

Comment:
Should we update this and other tests that use C# 9 to 10?

Reply:
> Should we update this and other tests that use C# 9 to 10?

Yes, I am planning to do this in a separate PR to reduce the noise.